summaries/summary-allev-inferred.json

```diff
@@ -3,316 +3,141 @@
     [
       3702,
       "Arabidopsis thaliana",
-      10004,
-      1131239,
+      8835,
+      772427,
       "annotations/taxid_3702/GO_annotations-3702-inferred-allev.tsv"
     ],
-    [
-      4528,
-      "Oryza longistaminata",
-      30,
-      30,
-      "annotations/taxid_4528/GO_annotations-4528-inferred-allev.tsv"
-    ],
```

Comment:
Any idea why several rice species like [this one](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4528) no longer produce any output? Is it that they no longer have gene ontology annotations?

Perhaps the annotations all belong now to _Oryza glaberrima_?

Reply:
As I learned, most of the rice species are still listed in Gene Ontology, but they may no longer be present in Gene Entrez, which we are using. This could happen for several reasons:

Merging with similar species: Some species may have been merged with more widely studied species, such as Oryza glaberrima, especially if they share a significant amount of genomic information. In such cases, the annotations might be consolidated into the primary species, causing the others to no longer appear separately.

Lack of recent annotations: Taxa may be removed from Gene Entrez if they haven't had any recent updates or annotations. If there is a lack of new genomic data or functional annotations, these species might be flagged for removal or archiving.

Data reclassification: Sometimes, data undergoes reclassification or a shift in how species are categorized, and some taxons may no longer meet the criteria for inclusion in Gene Entrez, leading to their removal.

In summary, while the annotations may still exist in Gene Ontology, the species may no longer appear in Gene Entrez either due to merging, lack of recent updates, or reclassification.

Reply:
FYI, I'm working with a time constraint and would really appreciate it if you could expedite the review of the pull request.

The only change I made in the process.ipynb file is in Cell 8, where I defined the list of 45 TaxIds available in Websurface.

Thanks for your understanding and for helping me review this quickly!